applications/asset_tracker_v2/src/cloud/aws_iot_integration.c

```diff
@@ -28,8 +28,8 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_CLOUD_INTEGRATION_LOG_LEVEL);
 #define BATCH_TOPIC_LEN (AWS_CLOUD_CLIENT_ID_LEN + 6)
 #define MESSAGES_TOPIC "%s/messages"
 #define MESSAGES_TOPIC_LEN (AWS_CLOUD_CLIENT_ID_LEN + 9)
-#define NEIGHBOR_CELLS_TOPIC "%s/ncellmeas"
-#define NEIGHBOR_CELLS_TOPIC_LEN (AWS_CLOUD_CLIENT_ID_LEN + 10)
+#define GROUNDFIX_TOPIC "%s/ground-fix"
```

Comment:
`GROUND_FIX` is the most common naming convention used throughout NCS. I think we should separate the words in the ATv2 implementation as well to be consistent with the rest of the tree.

Reply:
fixed